storm-server/src/test/java/org/apache/storm/container/oci/OciUtilsTest.java

```diff
@@ -117,22 +126,22 @@ public void adjustImageConfigForTopoTest() throws InvalidTopologyException {
         //case 3: configs are set properly; nothing will happen
         topoConf.put(Config.TOPOLOGY_OCI_IMAGE, image1);
         OciUtils.adjustImageConfigForTopo(conf, topoConf, topoId);
-        Assert.assertEquals(image1, topoConf.get(Config.TOPOLOGY_OCI_IMAGE));
+        assertEquals(image1, topoConf.get(Config.TOPOLOGY_OCI_IMAGE));
 
         //case 4: topology oci image is not set; will be set to default image
         topoConf.remove(Config.TOPOLOGY_OCI_IMAGE);
         OciUtils.adjustImageConfigForTopo(conf, topoConf, topoId);
-        Assert.assertEquals(defaultImage, topoConf.get(Config.TOPOLOGY_OCI_IMAGE));
+        assertEquals(defaultImage, topoConf.get(Config.TOPOLOGY_OCI_IMAGE));
 
         //case 5: any topology oci image is allowed
         allowedImages.add("*");
         String image2 = "storm/rhel7:dev_wow";
         topoConf.put(Config.TOPOLOGY_OCI_IMAGE, image2);
         OciUtils.adjustImageConfigForTopo(conf, topoConf, topoId);
-        Assert.assertEquals(image2, topoConf.get(Config.TOPOLOGY_OCI_IMAGE));
+        assertEquals(image2, topoConf.get(Config.TOPOLOGY_OCI_IMAGE));
     }
 
-    @Test(expected = WrappedInvalidTopologyException.class)
+    @Test
     public void adjustImageConfigForTopoNotInAllowedList() throws InvalidTopologyException {
```

Comment:
this method should not throw any exception

Reply:
done